### ISSUE TITLE
Update readme to match current codebase

### ISSUE THREAD

**Issue by thorncp** [thorncp/rubot#14]
The current readme is behind on reality.

**Comment by mculp** [comment]
Is it really? I wrote it not too long ago in terms of commits to rubot.

**Comment by mculp** [comment]
Oh, that was from 4 years ago. lol

**Comment by thorncp** [comment]
Yeah, I just updated the description based on #34. I didn't take the time to actually look at the readme.

**Comment by thorncp** [comment]
The previous description was "Documentation" which is not very useful...
